examples/export/utils.py

```diff
@@ -1,31 +0,0 @@
-import urllib, io, os
```

Comment:
This deletion appears unrelated to the PR. Could you either move this to a separate PR or expand the description of this PR?

Reply:
extended descriptions, utils.py is not needed anymore because if embed=True

Reply:
ok, makes sense.